paddle/phi/kernels/funcs/transpose_functor.h

```diff
@@ -27,161 +27,115 @@ enum { kTransposeMKLDNNFP32 = 1, kTransposeMKLDNNINT8 = 2 };
 
 enum PermuteType {
   kCopy = 1,
-  kTranspose = 2,
-  kVecPermute = 3,
-  kGeneralPermute = 4
+  kSwapTranspose = 2,
+  kGeneralTranspose = 3,
+  kVecPermute = 4,
+  kGeneralPermute = 5
 };
 
 constexpr int kBlockRows = 16;
 constexpr int kTileSize = 32;
+constexpr int kShareCol = (kTileSize + 1);
+
+#define GETTILESIZE(LEN_, ALIGN_) \
```

Comment:
`GETTILESIZE` -> `GET_TILE_SIZE`

Reply:
Parts of the pr suggestions have been estabulished in [PR33051](https://github.com/PaddlePaddle/Paddle/pull/33051)